Tuesdays/Week-Class/Inheratance/Assets/Scripts/UIControl.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class UIControl : MonoBehaviour{
```

Comment:
this controls specifically the HP UI give it.
call it "HpView"
controller less fits here, as we use  it in cases where we change the game state